src/shared/scrapers/US/MI/index.js

```diff
@@ -125,55 +154,62 @@ const scraper = {
       cases: 0
     };
 
+    const $trs = $table.find('tbody > tr');
     $trs.each((index, tr) => {
       const $tr = $(tr);
+      const rowData = {};
+
+      $tr.find('td').each((columnIndex, td) => {
+        const $td = $(td);
 
-      let cases = parse.number(parse.string($tr.find('> *:nth-child(2)').text()) || 0);
-      let deaths = parse.number(parse.string($tr.find('> *:last-child').text()) || 0);
-      const county = geography.getCounty(parse.string($tr.find('> *:first-child').text()), 'iso2:US-MI');
+        const key = dataKeysByColumnIndex[columnIndex];
+        rowData[key] = getValue(key, $td.text());
+      });
 
       // Remember these to add them to Wayne County instead
-      if (county === 'Detroit City') {
-        detroitCases = cases;
-        detroitDeaths = deaths;
+      // TODO: Handling of Detroit seems brittle.
+      if (rowData.county === DetroitCity) {
+        detroitCases = rowData.cases;
+        detroitDeaths = rowData.deaths;
       }
-      if (county === 'Wayne County') {
-        cases += detroitCases;
-        deaths += detroitDeaths;
+
+      if (rowData.county === WayneCounty) {
+        rowData.cases += detroitCases;
+        rowData.deaths += detroitDeaths;
       }
 
-      if (county === 'Out of State' || county === 'Other' || county === 'Not Reported' || county === 'Unknown') {
-        unassignedObj.cases += cases;
-        unassignedObj.deaths += deaths;
+      if (['Out of State', 'Other', 'Not Reported', 'Unknown'].includes(rowData.county)) {
+        unassignedObj.cases += rowData.cases;
+        unassignedObj.deaths += rowData.deaths;
         return;
       }
 
+      // TODO: What is this doing and is it legitimate?
       if (index < 1 || index > $trs.get().length - 2) {
         return;
```

Comment:
Is this not just to handle the header row and the totals row at the end?

Reply:
Yeah but seems to me like it’s ignoring the first county, and normally we exclude via css